WordPress/Classes/Utility/PushAuthenticationManager.swift

```diff
@@ -63,14 +63,17 @@ class PushAuthenticationManager {
             return
         }
 
-        showLoginVerificationAlert(message) { approved in
```

Comment:
I know this PR isn't final, but I think we'll need to leave code in here to show an alert if the app is currently open when the notification arrives? We might need to split this method into two parts – one we can reuse in the 'backgrounded app' case, and one which calls into that but also does the UI part in the case where the app is in the foreground?

Reply:
Good point about about the need for the alert if the app is open.  I'll make sure that functionality is retained.